tendrl/node_agent/node_sync/event_utils.py

```diff
@@ -0,0 +1,38 @@
+import json
+import os
+
+from tendrl.commons.utils import log_utils as logger
+from tendrl.commons.utils.time_utils import now as tendrl_now
+
+
+def emit_event(resource, curr_value, msg, instance,
```

Comment:
Should we move this function to commons and use in GI as well as node-agent?

Reply:
ack